### DISCUSSION TITLE
"default" datasource is sorted incorrectly

### DISCUSSION THREAD

**Discussion by robx** [grafana/grafana#38148] [Dashboards, Panels & Visualizations]
**What happened**:

When setting up a query, in the datasource dropdown, I switched away from the non-"default" datasource, and then couldn't find it at a reasonable place when trying to switch back. Concretely, the default datasource showed up in the alphabetically sorted list of datasources between two datasources whose names started with "r". It seems to appear right next to the actual datasource that's labelled "default".

![Screenshot 2020-09-04 at 10 42 37](https://user-images.githubusercontent.com/132113/92220431-48170b00-ee9c-11ea-9655-03b913020e0f.png)

**What you expected to happen**:

The "default" datasource should appear ideally at the top (if it's really special), or otherwise be sorted alphabetically with "d".

**How to reproduce it (as minimally and precisely as possible)**:

If you're inside Grafana Labs, create a dashboard on ops.

**Environment**:
- Grafana version: 7.1.5 (commit: 9893b8c53d, branch: HEAD)
- Data source type & version: the default datasource on this instance seems to be a graphite datasource
- User OS & Browser: macos / firefox

**Comment by hugohaggmark** [comment]
Thanks @robx, you can also search (by typing) for default but I understand that having it in the top would bring value :)

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
